presto-main/src/main/java/io/prestosql/operator/PagesIndex.java

```diff
@@ -493,20 +493,15 @@ public LookupSourceSupplier createLookupSourceSupplier(
             // This code path will trigger only for OUTER joins. To fix that we need to add support for
             //        OUTER joins into NestedLoopsJoin and remove "type == INNER" condition in LocalExecutionPlanner.visitJoin()
 
-            try {
-                LookupSourceSupplierFactory lookupSourceFactory = joinCompiler.compileLookupSourceFactory(types, joinChannels, sortChannel, outputChannels);
-                return lookupSourceFactory.createLookupSourceSupplier(
-                        session,
-                        valueAddresses,
-                        channels,
-                        hashChannel,
-                        filterFunctionFactory,
-                        sortChannel,
-                        searchFunctionFactories);
-            }
-            catch (Exception e) {
-                log.error(e, "Lookup source compile failed for types=%s error=%s", types, e);
```

Comment:
same in `io.prestosql.operator.PagesIndex#createPagesHashStrategy(java.util.List<java.lang.Integer>, java.util.OptionalInt, java.util.Optional<java.util.List<java.lang.Integer>>)` 
https://github.com/prestosql/presto/blob/e7199dba8e2551cd6d70cae891c8e2e14200a678/presto-main/src/main/java/io/prestosql/operator/PagesIndex.java#L436-L442
?